From 5f1b8786bd9ff513e4d8d73308e13a62969559c5 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:08:52 -0400
Subject: [usb] usb_wwan: remove some superfluous comments

Message-id: <1410887463-20674-461-git-send-email-dzickus@redhat.com>
Patchwork-id: 95611
O-Subject: [RHEL7 PATCH 460/591] USB: usb_wwan: remove some superfluous comments
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 7d5dddda088ed054f1519d03f997346c03bfc778
Author: Johan Hovold <jhovold@gmail.com>
Date:   Mon May 26 19:23:29 2014 +0200

    USB: usb_wwan: remove some superfluous comments

    Remove some more outdated or superfluous comments.

    Signed-off-by: Johan Hovold <jhovold@gmail.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/serial/usb_wwan.c b/drivers/usb/serial/usb_wwan.c
index 5042faa..c951f75 100644
--- a/drivers/usb/serial/usb_wwan.c
+++ b/drivers/usb/serial/usb_wwan.c
@@ -178,7 +178,6 @@ int usb_wwan_ioctl(struct tty_struct *tty,
 }
 EXPORT_SYMBOL(usb_wwan_ioctl);
 
-/* Write */
 int usb_wwan_write(struct tty_struct *tty, struct usb_serial_port *port,
      const unsigned char *buf, int count)
 {
@@ -429,7 +428,6 @@ void usb_wwan_close(struct usb_serial_port *port)
 
  portdata = usb_get_serial_port_data(port);
 
- /* Stop reading/writing urbs */
  spin_lock_irq(&intfdata->susp_lock);
  portdata->opened = 0;
  if (--intfdata->open_ports == 0)
@@ -454,7 +452,6 @@ void usb_wwan_close(struct usb_serial_port *port)
 }
 EXPORT_SYMBOL(usb_wwan_close);
 
-/* Helper functions used by usb_wwan_setup_urbs */
 static struct urb *usb_wwan_setup_urb(struct usb_serial_port *port,
           int endpoint,
           int dir, void *ctx, char *buf, int len,
@@ -467,7 +464,6 @@ static struct urb *usb_wwan_setup_urb(struct usb_serial_port *port,
  if (!urb)
   return NULL;
 
- /* Fill URB using supplied data. */
  usb_fill_bulk_urb(urb, serial->dev,
      usb_sndbulkpipe(serial->dev, endpoint) | dir,
      buf, len, callback, ctx);
@@ -567,7 +563,6 @@ static void stop_urbs(struct usb_serial *serial)
  struct usb_serial_port *port;
  struct usb_wwan_port_private *portdata;
 
- /* Stop reading/writing urbs */
  for (i = 0; i < serial->num_ports; ++i) {
   port = serial->port[i];
   portdata = usb_get_serial_port_data(port);
@@ -648,11 +643,9 @@ int usb_wwan_resume(struct usb_serial *serial)
 
  spin_lock_irq(&intfdata->susp_lock);
  for (i = 0; i < serial->num_ports; i++) {
-  /* walk all ports */
   port = serial->port[i];
   portdata = usb_get_serial_port_data(port);
 
-  /* skip closed ports */
   if (!portdata || !portdata->opened)
    continue;
 
-- 
1.7.1